project.md

```diff
@@ -5,47 +5,65 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+	This DSL is aimed for those who want to create sheet music and chord progressions or melodies quickly, and who don't have that much experience with programming.
+
 ### What's the need?
 
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+	I'm aiming to create a program that would let them quickly prototype things, without having to go through all the hurdles of designing things in a full music production application.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+	A language lets you design these fast, in a way that's easy to edit, and in a way that you can easily export to different forms using existing libraries. This helps to make it a quick way to prototype melodies and such, rather than having to use a graphical representation of notes that may be slower. (built in repetition and ways to vary it, especially, should be an good way to specificaully work towards this need.)
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+	I came up with it as a good project that allows me to design something novel, with unique skills added. It also lets me design a project about a domain I already have a little bit of experience in.
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+	Producing sheet music or melodies
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+	A user interacts with this language by writing a new song in plaintext. The user would define a number of chords and melodies as a collection of notes, then would put together these chords and melodies in increasinly complicated ways to eventually lead to a final song. This lets you define a structure and then use it repeatedly, which should make it quicker to edit structures used across the program.
```

Comment:
So essentially, the user builds their own language library by creating chords and melodies? Seems pretty DSL-y to me.

Reply:
I also wanted to note that I was thinking of doing a similar feature in my own DSL, which is a neat coincidence! Particularly, taiko has many beat patterns that will reoccur across different music (horse beat, for example).